### PR TITLE
Update-Readme.md

### PR DESCRIPTION
**Pull request by Patfarmurs**

### CODE REVIEW

**Review by ehmaddd — APPROVED** [review]
Done great effort @Patfarmurs 

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f63454f444766654f594d52784b2f67697068792e676966


You can merge it now

File: README.md [hunk 1]
```diff
@@ -1,70 +1,177 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+- Original design idea by [Murat Korkmaz on Behance](https://www.behance.net/muratk).
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/jonas-45/motorcycle-booking-react/blob/LICENSE.md) licensed.
```
Comment (Jawad-Nazari) [inline README.md:175]: Hi @Patfarmurs ,
The link to the license file is not working. Please update it.
image

File: README.md [hunk 2]
```diff
@@ -1,70 +1,177 @@
...
+
+- Github: [GitHub](https://github.com/ehmaddd/)
+- Twitter: [Twitter](https://twitter.com/ehmaddd_pk)
+- Linkedin: [LinkedIn](https://www.linkedin.com/in/ehmaddd/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭Future Features <a name="future-features"></a>
+
+
```
Comment (Jawad-Nazari) [inline README.md]: Please , add something for future features like improve style and add more functionality .currently is totally empty.

**Review by Jawad-Nazari — CHANGES_REQUESTED** [review]
Hi @Patfarmurs ,
You have almost done to finish this activity. The link to the license is broken please update it. Also please something in future feature as currently is empty.

**Pull request merged**